scripts/release.sh

```
@@ -0,0 +1,343 @@
+
+if [[ "$SKIP_VERSION_UPDATE" == "false" ]]; then
+  # Version update
+  echo ""
+  echo "🔧 Setting version to $VERSION..."
+  run "npm version \"$VERSION\" --no-git-tag-version"
+
+  # README update
+  echo ""
+  echo "📝 Updating version in README.md..."
+  # Update version references in code examples using extended regex for precise semver matching
+  run "sed -i '' -E 's/@[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9]+\.[0-9]+)?(-[a-zA-Z0-9]+\.[0-9]+)*(-[a-zA-Z0-9]+)?/@'"$VERSION"'/g' README.md"
```

Shellcheck (0.10.0)

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)